backend/app/assets/stylesheets/spree/backend/components/_pills.scss

```diff
@@ -0,0 +1,43 @@
+.pill {
+  display: inline-block;
+  padding: 2px 10px;
+  border-radius: 10px;
+  font-size: 11px;
```

Comment:
I like us to move forward to relative units all over the place. That’s the way all frontend frameworks went for a reason. 

I see the problem with the small base font size we have. So, I won’t insist to make this change right now. 

But, could you please make this font size a variable then?